app/api/login/route.ts

```diff
@@ -37,9 +37,10 @@ export async function POST(request: NextRequest) {
         }
 
         const response = NextResponse.json(userObj);
+        const maxSessionAge = payload.exp - payload.iat;
         await createSession({
             envelope: body.envelope, 
-            exp: payload.exp, 
+            exp: maxSessionAge, 
```

Comment:
This field/param is used to set both the `exp` claim in the session value, (which should be the same) and also the `maxAge` property on the cookie. So we can't just make this `maxAge` becuase then we will be breaking the `exp` claim on the token itself.